dev-docs/RFCs/v5.0/viewport-animation-rfc.md

```diff
@@ -2,18 +2,69 @@
 
 * **Authors**: Ravi Akkenapally
 * **Date**: Aug 2017
-* **Status**: Placeholder
+* **Status**: **Draft**
 
 
-## Proposal: Viewport View Matrix Decomposition support
```

Comment:
@ibgreen should I create a new RFC with this content ?

Reply:
Just put it in an issue for now

Reply:
https://github.com/uber/deck.gl/issues/940